_posts/2020-03-06-ok-mauro-but-i-want-to-do-paging-and-sorting.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+header_image: /img/posts/ok-mauro-but-i-want-to-do-paging-and-sorting/header.jpg
+title: "OK Mauro, but I want to do paging AND sorting"
+author: Mauro Servienti
+synopsis: "Paging and sorting can be easily applied in isolation, even in distributed systems. When in need of paging AND sorting, the level of coordination required grows exponentially. It turns out that the problem is not that different from searching in distributed systems, and similar techniques can be used to address the paging and sorting dilemma."
+tags:
+- SOA
+- Services ViewModel Composition
+category: view-model-composition
+---
+
+It's a kind of [tornado of souls](https://www.youtube.com/watch?v=K-HzFACAedk) here. Coronavirus is all around where I live, and smart working, as Italians call remote working, is all the rage. Articles, posts, and twitter threads are all about how to be great at remote working.
```

Comment:
Added a highlight because it wasn't a familiar term for me and it read a bit weird the first time.

```suggestion
It's a kind of [tornado of souls](https://www.youtube.com/watch?v=K-HzFACAedk) here. Coronavirus is all around where I live, and _smart working_, as Italians call remote working, is all the rage. Articles, posts, and twitter threads are all about how to be great at remote working.
```